src/backend/pipeline/cont_analyze.c

```diff
@@ -2183,11 +2183,14 @@ proj_and_group_for_windows(SelectStmt *proc, SelectStmt *view, ContAnalyzeContex
 	}
 
 	/* Create and truncate projection and grouping on temporal expression */
+	context->hoisted_name = ARRIVAL_TIMESTAMP;
```

Comment:
For some reason I though we were rejecting `arrival_timestamp` as a column name or alias, which we probably should.